charts/codesealer/values.yaml

```diff
@@ -228,9 +228,9 @@ worker:
       # 'TLS1.0', 'TLS1.1', 'TLS1.2', 'TLS1.3'
       # If this option is not set, TLS 1.2 is used as the default.
       tlsMinVersion: "TLS1.2"
-      # Paths for TLS certificate and key files that are used if 'network.tls' is set to true.
-      # When using the Management Portal, this certificate and key will be used as fallback
-      # when *no* certificate and key is specified in the Management Portal.
+      # Paths for TLS certificate and key files that are used if 'manager.tls' is set to true.
+      # When using the Management Portal, the certificate and key above will be used as
+      # fallback when *no* certificate and key is specified in the Management Portal.
```

Comment:
Revert this (or update to the latest wording on master).

Reply:
Changes made